ethcore/src/snapshot/service.rs

```diff
@@ -220,7 +221,7 @@ pub struct ServiceParams {
 	/// Usually "<chain hash>/snapshot"
 	pub snapshot_root: PathBuf,
 	/// A handle for database restoration.
-	pub db_restore: Arc<DatabaseRestore>,
+	pub client: Arc<Client>,
```

Comment:
It's contradictory to @0x7CFE refactor of `ethcore`. We don't want any modules to require `Client`, but only the interface that is actually being used